section3/ex34.rb

```diff
@@ -0,0 +1,26 @@
+animals = ['bear', 'ruby', 'peacock', 'kangaroo', 'whale', 'platypus']
+# The animal at 1.
+# * The animal at index 1 is 'ruby' and it's the second animal.
+# The third (3rd) animal.
+# The 3rd animal is at index 2 and is peacock.
+# The first (1st) animal.
+# The first animal is at index 0 and is bear.
+# The animal at 3.
+# The animal at 3 is the 4th animal and is kangaroo.
+# The fifth (5th) animal.
+# The 5th animal is at index 4 and is whale.
+# The animal at 2.
+# The animal at index 2 is the 3rd animal and is peacock.
+# The sixth (6th) animal.
+# The 6th animal is at index 5 and is platypus.
+# The animal at 4.
+# The animal at index 4 is the 5th animal and is whale.
+
+puts "#{animals[1]}"
+puts "#{animals[2]}"
+puts "#{animals[0]}"
+puts "#{animals[3]}"
+puts "#{animals[4]}"
+puts "#{animals[2]}"
+puts "#{animals[5]}"
+puts "#{animals[4]}"
```

Comment:
same feedback from section1—no need to wrap these with string interpolation since you aren't combining them with other strings.